js_modules/dagster-ui/packages/ui-core/src/app/time/ThemeSelect.tsx

```diff
@@ -11,14 +11,27 @@ export const ThemeSelect = ({theme, onChange}: Props) => {
     {
       key: DagsterTheme.Light,
       label: 'Light',
+      icon: 'sun',
     },
     {
       key: DagsterTheme.Dark,
       label: 'Dark',
+      icon: 'nightlight',
+    },
+    {
+      key: DagsterTheme.LightHighContrast,
+      label: 'Light (red/green deficiency)',
+      icon: 'sun',
+    },
+    {
+      key: DagsterTheme.DarkHighContrast,
+      label: 'Dark (red/green deficiency)',
```

Comment:
Just checking: should this maybe be `Dark, no red/green` or something? I guess I don't know if it will be clear enough what `red/green deficiency` will mean.

Reply:
Yeah I think we should use the technical term ? 
<img width="695" alt="Screenshot 2025-02-06 at 2 07 38 PM" src="https://github.com/user-attachments/assets/40dc85ea-79ee-40d1-bfc6-586e32ae8800" />

Really wish we would let users change colors themselves too just because the spectrum is so wide